Documentation/manualtest-guide.md

```diff
@@ -0,0 +1,12 @@
+Manual Tests Instruction
+========================
+Test Name: WindowsAuthentication_RoundTrips_Echo
+Set Up Requirement
+1. Must run on a domain machine.
+
+Steps
```

Comment:
Formatting nits: 

You need to add a line break between your list heading and your ordered list, e.g. 

```
Set Up Requirement
1. Step 1
2. Step 2
```
shows

Set Up Requirement
1. Step 1
2. Step 2

and 

```
Set Up Requirement

1. Step 1
2. Step 2
```
shows


Set Up Requirement

1. Step 1
2. Step 2

(note the indentation)